aws/resource_aws_apigatewayv2_integration.go

```diff
@@ -311,11 +338,33 @@ func resourceAwsApiGatewayV2IntegrationUpdate(d *schema.ResourceData, meta inter
 		for k, v := range nop {
 			variables[k] = v
 		}
+
 		req.RequestParameters = variables
 	}
 	if d.HasChange("request_templates") {
 		req.RequestTemplates = stringMapToPointers(d.Get("request_templates").(map[string]interface{}))
 	}
+	if d.HasChange("response_parameters") {
+		o, n := d.GetChange("response_parameters")
+		os := o.(*schema.Set)
+		ns := n.(*schema.Set)
+		del := os.Difference(ns).List()
```

Comment:
I think has go common enough at this point, should we maybe encapsulate it into a function and reuse? or is it too simple?

Reply:
Something like we do for `diffStringMaps`, for example:

```go
func diffSets(oldSet, newSet *schema.Set) (create, remove, unchanged *schema.Set) {
...
}
```

Reply:
yes, i believe there something similar for maps already but i dont recall where.